week-5/B-promises/exercise.js

```diff
@@ -0,0 +1,156 @@
+/*
+  Callbacks and Promises
+  ---------------------------------
+  Open index.html in your browser
+*/
+
+/*
+  EXERCISE 1
+  =======
+
+  Write a function that inserts your name into the #exercise1 element when
+  the button is clicked
+*/
+document.querySelector('#button1')
+  .addEventListener('click', exercise1)
+
+function exercise1() {
+  // Write your implementation here
+}
+
+/*
+  EXERCISE 2
+  =======
+
+  Write a callback function that inserts the result into the #exercise2
```

Comment:
I think some documentation is needed to explain how this callback gets called.

Reply:
Wasn't clear to me until I scrolled down and looked at the function definition for functionThatCallsBack